test/endpoints/getResult.test.ts

```diff
@@ -310,29 +310,33 @@ describe('ingress API request headers', () => {
     fetchSpy.mockRestore()
   })
 
-  test('req headers are the same (except Cookie) when no proxy secret', async () => {
+  test('req headers are the same (except proxy headers) when no proxy secret', async () => {
```

Comment:
Clarity suggestion

```suggestion
  test('Fpjs-Proxy-Client-Ip and Fpjs-Proxy-Forwarded-Host headers are added to the proxied request headers even if proxy secret is undefined ', async () => {
```

Reply:
@JuroUhlar I want to prevent the creation of new tests for the potential new proxy headers. Mentioning only **Client-IP** and **Forwarded-Host** might push the developer to create a new test instead of editing this one. I could be wrong, but do you have any idea about this? To make it clear, should we use `FPJS-Proxy-Client-IP and FPJS-Proxy-Forwarded-Host headers` or `FPJS-Proxy headers` / `Proxy headers`? We can also check this in the reviews whether the developers have updated this test. Both options seem fine to me. We can proceed based on your guidance 😇 

> Additionally, this test checks whether the custom headers are preserved. We'll need to mention this in the test name as well

Reply:
Sure,  you can say something like
```suggestion
  test('Even if proxy secret is undefined, other Fpjs-Proxy-* headers are still added to the proxied request headers. Original headers are preserved.', async () => {
```